notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>Please add a full link to the paper: https://doi.org/...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>This will be resolved once the paper is added to references.bib in EDS book core repo (see for instance, https://github.com/alan-turing-institute/environmental-ds-book/blob/d6c95fd4517ff2e29c0dc89dd22661156584c1cc/book/_bibliography/references.bib#L332).</p>